content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md

```diff
@@ -269,7 +281,99 @@ This returns the same output as checking the flow status on Swagger UI. You see:
 
 If you have any questions about the Corda Node CLI commands, run the `--help` command for more information.
 
+## Set up and test the UI
+
+The Solar System CorDapp comes with a built-in UI. See the <a href="https://github.com/corda/samples-kotlin-corda5/tree/main/Tutorial/solarsystem/Web-UI">`Web-UI`</a> folder of the sample folder to study the code.
+
+### Before you start
+
+Before you can build the Solar System CorDapp UI, you must:
+
+* [Deploy your CorDapp to a local Corda 5 network](*deploy-the-cordapp-using-corda-cli)
+* Download the [Node.js](https://nodejs.org/en/download/) asynchronous event-driven JavaScript runtime for your platform.
+* Ensure that port `3000` is not being used by any other applications.
+
+{{< note >}}
+The UI proxy is hard-coded to run on port `3000`. A proxy is used because the current implementation of the web server does not populate any of the CORS headers. The proxy ensures that when the Open API is used by externally-hosted websites, no errors occur on the client side.
```

Comment:
@emisrz Expanded this note a bit. Can you check for accuracy?

Reply:
Perfect, all good 👍🏼